src/main.cpp

```diff
@@ -985,11 +987,12 @@ bool AcceptToMemoryPool(CTxMemPool &pool,
     TransactionClass allowedTx)
 {
     std::vector<COutPoint> vCoinsToUncache;
+    bool isRespend;
     bool res = AcceptToMemoryPoolWorker(pool, state, ptx, fLimitFree, pfMissingInputs, fOverrideMempoolLimit,
```

Comment:
shouldn't isRespend be set to false intially?

Reply:
it's set to false in AcceptToMemoryPoolWorker